instrumentation/lettuce-5.0/src/main/java/io/lettuce/core/protocol/CommandHandler.java

```diff
@@ -0,0 +1,30 @@
+package io.lettuce.core.protocol;
+
+import com.newrelic.api.agent.Trace;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+
+import io.netty.channel.ChannelHandlerContext_Instrumentation;
+//import io.netty.channel.ChannelHandlerContext_Instrumentation;
+import io.netty.channel.ChannelPromise;
+
+@Weave
+public abstract class CommandHandler {
```

Comment:
Same comment about naming weaved library classed with `_Instrumentation` and specifying the `originalName` in the `Weave` annotation.

Reply:
ok, did this in 6 but didn't do in older versions...will do